cache_images/cloud.yml

```diff
@@ -183,30 +183,30 @@ provisioners:
     - type: 'shell'
       inline:
         - 'set -e'
-        - 'mkdir -p /tmp/automation_images'
+        - 'mkdir -p /var/tmp/automation_images'
```

Comment:
Actually, this will (I think) have a knock-on effect:

All CI VM images will include a copy of the instructions/scripts used to build them.  I just checked, the `finalize()` functions don't touch `/var/tmp/`, only `/tmp`.

Reply:
I don't understand this comment but I think it is addressed by my "undo the rm /var/tmp/auto*" push just now. If it doesn't, please let me know.

Reply:
Ya, you got it.